packages/preprocess/src/svelte-ts-preprocess.ts

```diff
@@ -151,7 +151,7 @@ export function createPreprocessOptions(opts?: Partial<PreprocessOptions>): Prep
   return {
     compilerOptions: opts.compilerOptions ? opts.compilerOptions : defaultCompilerOptions,
     env: opts.env ? opts.env : createEnv(),
-    showErrors: opts.showErrors ? opts.showErrors : true
+    hideErrors: !!opts.hideErrors
```

Comment:
I think `hideErrors` no need here

Reply:
It has to be passed down trough that function doesn't it?